docs/site/translation/number-currency-formats/number-and-currency-patterns.md

```diff
@@ -128,3 +128,16 @@ X digit-two
 
 Xdigit-other
 
+## Rational Formatting
+
+There are special patterns for formatting rational fractions, such as in the following:
+
+![Screenshot 2025-03-29 at 17 46 03](https://github.com/user-attachments/assets/1a5a3ba5-e38e-49f5-9219-b1cd1d35ad08)
```

Comment:
this is an ephemeral URL. this file needs to be brought into the repo (in this same directory is fine)